include/linux/cpu.h

```diff
@@ -2,14 +2,14 @@
 /*
  * include/linux/cpu.h - generic cpu definition
  *
- * This is mainly for topological representation. We define the
- * basic 'struct cpu' here, which can be embedded in per-arch
+ * This is mainly for topological representation. We define the 
```

Comment:
This trailing whitespace is present upstream, this just reverts it to the in tree style.